apps/website/src/components/content/ProfileCard.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+import React from "react";
+import type { Ambassador } from "@alveusgg/data/src/ambassadors/core";
+import Image, { type ImageProps } from "next/image";
+import type { AmbassadorImage } from "@alveusgg/data/src/ambassadors/images";
+import type { StaffImage } from "./LinkHover";
+import Heading from "./Heading";
+
+export type ProfileCardProps = {
+  name: string;
+  position?: string;
+  species?: Ambassador["species"];
+  img?: AmbassadorImage | ImageProps | StaffImage;
+  imgs?: false | AmbassadorImage[];
+  enclosure?: string;
+  isEnclosure?: boolean;
+  upwards: boolean;
+};
+
+type SpanProps = {
+  title: string;
+  titleName?: Ambassador["species"] | string;
+};
+
+const Span: React.FC<SpanProps> = ({ title, titleName }) => {
+  return (
+    <div className="inline text-yellow-300">
+      <span className="font-semibold">{title}:</span> {titleName}
+    </div>
+  );
+};
+
+const staffStyle = `absolute z-50 flex flex-col lg:flex-row  box-border  max-w-[448px] gap-4 items-center rounded bg-alveus-green-900 p-4 shadow-lg shadow-alveus-green-800`;
+
+const ambassadorStyle = `absolute z-50 flex flex-col xl:flex-row  -translate-x-1/4 -mt-[25px] items-center gap-4 max-w-[340px]  rounded  bg-alveus-green-900 p-4 shadow-lg shadow-alveus-green-800`;
+
+const enclosureStyle = `absolute z-50 flex flex-col -translate-x-1/4 -mt-[25px] items-center gap-4  rounded  bg-alveus-green-900 p-4 shadow-lg shadow-alveus-green-800`;
+
+const ProfileCard: React.FC<ProfileCardProps> = ({
+  name,
+  img,
+  imgs,
+  species,
+  enclosure,
+  isEnclosure,
+  position,
+  upwards,
+}) => {
+  const ambassadorStyles = `${ambassadorStyle} ${
+    upwards ? "-translate-y-full" : "mt-4"
+  } `;
```

Comment:
Please use the `classes` util for multiple classes, rather than relying on string interpolation